pyproject.toml

```diff
@@ -18,7 +18,7 @@ readme = "README.md"
 python = ">=3.9,<4.0"
 qdrant-client = "^1.9.1"
 pydantic = "^2.7.3"
-openai = "^1.33.0"
```

Comment:
Hey, Is there a reason to change this?

Reply:
Ah I think I know why this happened, actually `google-generativeai` raised an error which was fixed by updating `protobuf` (which updated `openai`) but after I made `google-generativeai` optional, I dowgraded `openai` to the earlier version and that's where this changed.

Reply:
So can you make it this version of higher meaning doing this: `^1.33.0` ?

Reply:
Yea, sorry for that 😅. its fixed now.

Reply:
Thanks!!